ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSchedule.java

```diff
@@ -28,10 +28,8 @@
 
   public static final BigInteger DEFAULT_CHAIN_ID = BigInteger.ONE;
 
-  public static ProtocolSchedule create() {
-    return fromConfig(
-        GenesisConfigFile.mainnet().getConfigOptions(), PrivacyParameters.DEFAULT, false);
-  }
+  public static final ProtocolSchedule DEFAULT =
```

Comment:
Changing from a static method to a constant is inessential to Type Transactions and has a long reach, adding many unneeded files and diff lines.